parm/config/config.resources

```diff
@@ -206,13 +229,21 @@ elif [ ${step} = "aeroanlfinal" ]; then
 
 elif [ ${step} = "anal" ]; then
 
-    export wtime_anal="01:00:00"
-    export npe_anal=1000
+    export wtime_anal="00:50:00"
+    export wtime_anal_gfs="00:40:00"
```

Comment:
I understand the granularity, but 10m is hardly any thing. So would keeping `gfs` the same as `gdas` have meaningful impact?

Reply:
In development, no. We'll just need to revisit these walltimes for GFSv17 ops (these are the current ops walltimes). Want me to change the `gfs` value to be 50mins?